rfd/0109-opsgenie-plugin.md

```diff
@@ -0,0 +1,109 @@
+# OPSGenie Integration RFD
+## Required approvers
+
+Engineering:
+Product:
+Security:
+
+## What
+
+This RFD proposes a plugin that allows Teleport to integrate with OpsGenie. This plugin will differ from existing plugins by being part of the Teleport binary directly.
+
+## Success criteria
+Users are able to configure teleport to automatically create alerts in OpsGenie from access requests.
+Alerts were chosen over incidents in OpsGenie for high priority alerts that indicate a service interruption.
+Users are also able to configure auto approval flows to be met under certain conditions. E.g when a requester is on-call 
+
+## Configuration UX
+
+The plugin will be configured using a toml file containing the required to interact with both Teleport access and the OpsGenie API.
```

Comment:
If the plugin is part of the Teleport binary, why did we decide to use a separate TOML file as opposed to the existing teleport.yaml configuration file?